### DISCUSSION TITLE
How to preserve kinematic tree structure when exporting USD files

### DISCUSSION THREAD

**Discussion by jiangranlv** [google-deepmind/mujoco#2240] [Asking for Help]
### Intro

Hi!

I am a (PhD student) at Peking University, I use MuJoCo for my research on robotics.

### My setup

MuJoCo 3.1.x

### My question

When using MuJoCo USDExporter, i found the kinematic tree is lost in the usd file. Can we have some easy solution to preserve it so that i can use this usd for simulation in isaac, not only rendering in isaac.

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
Currently, no. The USD exporter is visual only.
